inc/saihostintf.h

```diff
@@ -556,6 +556,9 @@ typedef enum _sai_hostif_packet_attr
     /** Ingress LAG [sai_object_id_t] (for receive-only) */
     SAI_HOSTIF_PACKET_INGRESS_LAG,
 
+    /** Ingress VLAN [sai_vlan_id_t] (for receive-only) */
+    SAI_HOSTIF_PACKET_INGRESS_VLAN,
```

Comment:
If it is the VLAN tag in the received packet, then the tag should be in the packet raw data. The application should be able to parse the packet to get the vlan tag. 

If it is PVID for untagged packet, then because the APP knows which incoming port the packet is coming from so it should know the PVID of that port.

It looks like the information can either be derived from the packet data or ASIC configuration. Is this true?